templates/socat.service.j2

```diff
@@ -1,8 +1,10 @@
 [Unit]
 Description=socat tunnel service {{ socat_service_name }}
-After=network.target
```

Comment:
I tested the role on the new Debian 11.2
There is `network-online.target`, not `network.target`.
You may have to do get_facts and insert the necessary units depending on the OS ...

Reply:
so debian prior 11 didn't support `network-online.target` and only had `network.target`? that's good to know! thx!

Reply:
just read into that and learned something new ;)